### PR TITLE
Remove send log buttons

### LINKED ISSUE #45: Send Logs does nothing
The "Send Logs" option in the app does nothing since we removed the `LogReporter` in https://github.com/jellyfin/jellyfin-androidtv/pull/9. We should evaluate if there is a way to do this without depending on a centralized service. If not the UI for this should be removed.

### PR DESCRIPTION
**Pull request by BnMcG**
Resolves #45 

Remove "send log" buttons that no longer work. Tested on Fire TV Gen 2. Removes buttons from both the login and the home screens.

### CODE REVIEW

File: app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java [hunk 1]
```diff
@@ -410,9 +397,6 @@ public void onItemClicked(Presenter.ViewHolder itemViewHolder, Object item,
                         Intent settings = new Intent(getActivity(), SettingsActivity.class);
                         getActivity().startActivity(settings);
                         break;
-                    case REPORT:
-                        Utils.reportError(getActivity(), "Send Log to Dev");
```
Comment (cvium) [inline app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java]: Should this function also be removed?

Comment (thornbill) [reply app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java]: I'm fine with it either way. That method just handles displaying a modal confirmation message, so we may reuse it.

File: app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java [hunk 2]
```diff
@@ -365,20 +363,9 @@ protected void addAdditionalRows(ArrayObjectAdapter rowAdapter) {
         toolsRow = new ArrayObjectAdapter(mGridPresenter);
         toolsRow.add(new GridButton(SETTINGS, mApplication.getString(R.string.lbl_app_settings), R.drawable.gears));
         toolsRow.add(new GridButton(LOGOUT, mApplication.getString(R.string.lbl_logout) + TvApp.getApplication().getCurrentUser().getName(), R.drawable.logout));
-
-        sendLogsButton = new GridButton(REPORT, mApplication.getString(R.string.lbl_send_logs), R.drawable.upload);
```
Comment (thornbill) [inline app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java]: Can we remove the `sendLogsButton` variable entirely?

File: app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java (hunk 1 above)
Comment (BnMcG) [reply app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java]: I left it in as it's referenced elsewhere - I'm happy to remove it though.

File: app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java (hunk 2 above)
Comment (BnMcG) [reply app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragment.java]: Derp. I've now removed the instance variable for sendLogsButton, and also the "REPORT" constant. Android Studio also suggested converting toolsRow to a local variable, so I made that change, too.

**Pull request merged**